cron/monthly/user-report.js

```diff
@@ -198,25 +196,32 @@ const processBacker = async FromCollectiveId => {
         createdAt: { [Op.gte]: startDate, [Op.lt]: endDate },
         SubscriptionId: { [Op.ne]: null },
       },
+      deletedAt: null,
     },
-    include: [{ model: models.Subscription }],
-  });
-  const ordersByCollectiveId = {};
-  orders.map(o => {
-    ordersByCollectiveId[o.CollectiveId] = o;
+    include: [
+      {
+        model: models.Subscription,
+        where: {
```

Comment:
Adding a where clause to the include automatically makes it required. I think we may miss orders that don't have a subscription here.

Reply:
Hum, I'll better use just a filter then. thanks for the good catch.